int32 HOPE = 0
int32 FEAR = 1
int32 RESET = 2
std_msgs/Int32 type